CHANGELOG.md

```diff
@@ -3,6 +3,26 @@
 Details changes in each release of EarlGrey. EarlGrey follows
 [semantic versioning](http://semver.org/).
 
+## [1.5.0](https://github.com/google/EarlGrey/tree/1.5.0) (10/31/2016)
+
+```
+Baseline: [d46a15f]
+   + [d46a15f]: Update Info.plist / Podspec / Cheatsheet for EarlGrey 1.4.0
+```
+
+### Compatibility
+* Add iOS 10 Support to Travis tests.
```

Comment:
You should change this to:

Compatibility
* Requires iOS 8 as the minimum deployment target.
* The EarlGrey gem runs out of the box for Swift 3.0 and Swift 2.3. Swift 2.2 is no longer supported.
* Has been tested for support till iOS 10.01 on devices and simulator.